ci-operator/jobs/openstack-k8s-operators/neutron-operator/openstack-k8s-operators-neutron-operator-master-presubmits.yaml

```diff
@@ -406,3 +406,51 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )neutron-operator-build-deploy-kuttl,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build02
+    context: ci/prow/precommit-check
+    decorate: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openstack-k8s-operators-neutron-operator-master-precommit-check
+    optional: true
```

Comment:
should precommit be optional? 

in my opinion it should not be but we should agree about this and be consistent across all the operator repos

Reply:
Ok. Marking it as voting. 

`golangci-lint` is failing, sometimes it passes and sometimes it fails.